### PR TITLE
Added new ESP8266 Beyond Robotix Kahuna

### PR DESCRIPTION
**Pull request by SamJCKnox**
We have developed new hardware and software to support the ESP8266 telemetry modules. Our software is open-source [here](https://github.com/BeyondRobotix/Kahuna) and upgrades the ESP GUI. We have compiled comprehensive documentation on our [website](https://www.beyondrobotix.com/products/kahuna) that should make using WiFi with any autopilot much more accessible. We are avid Ardupilot enthusiasts and hope to help contribute to this great cause.

### CODE REVIEW

File: common/source/docs/common-esp8266-telemetry.rst [hunk 1]
```diff
@@ -7,12 +7,15 @@ ESP8266 wifi telemetry
...
+.. image:: ../../../images/Kahuna_ESP8266.jpg
+    :target: ../_images/Kahuna_ESP8266.jpg
+
 The ESP8266 wifi module is an inexpensive programmable wifi modules available from many sources.  It also comes included with most :ref:`Pixracers <common-pixracer-overview>`.
 
 Where to Buy
 ------------
-
-The `adafruit HUZZAH ESP8266 breakout board <https://www.adafruit.com/product/2471>`__ is recommended because it can be connected to an autopilot without the need for 3.3V regulator or level shifters.
+- The `Beyond Robotix - Kahuna <https://www.beyondrobotix.com/products/kahuna>`__ is a ready to use ESP8266 module that connects directly to an autopilot. It is pre-flashed with ardupilot firmware, connects directly to the Pixhawk range of products and has a u.fl connector for an external antenna. Further documentation for this unit is found `here <https://docs.google.com/document/d/1VyOsp9_q6dIAdYdWuDFcWoqqrNy_vbFMANubZA3Uz5g/edit?usp=sharing>`__.
```
Comment (khancyr) [inline common/source/docs/common-esp8266-telemetry.rst:17]: should be after adafruit one
remove pixhawk for autopilot
It isn't ardupilot firmware but a fork

File: common/source/docs/common-esp8266-telemetry.rst [hunk 2]
```diff
@@ -14,8 +14,8 @@ The ESP8266 wifi module is an inexpensive programmable wifi modules available fr
 
 Where to Buy
 ------------
-- The `Beyond Robotix - Kahuna <https://www.beyondrobotix.com/products/kahuna>`__ is a ready to use ESP8266 module that connects directly to an autopilot. It is pre-flashed with ardupilot firmware, connects directly to the Pixhawk range of products and has a u.fl connector for an external antenna. Further documentation for this unit is found `here <https://docs.google.com/document/d/1VyOsp9_q6dIAdYdWuDFcWoqqrNy_vbFMANubZA3Uz5g/edit?usp=sharing>`__.
 - The `adafruit HUZZAH ESP8266 breakout board <https://www.adafruit.com/product/2471>`__ can be connected to an autopilot without the need for 3.3V regulator or level shifters.
+- The `Beyond Robotix - Kahuna <https://www.beyondrobotix.com/products/kahuna>`__ is a ready to use ESP8266 module that connects directly to an autopilot. The Kahuna is pre-flashed with a fork of ardupilot firmware, has a JST-GH connector to connect easily to several off-the-shelf autopilots and has a u.fl connector for an external antenna. Further documentation for this unit is found `here <https://docs.google.com/document/d/1VyOsp9_q6dIAdYdWuDFcWoqqrNy_vbFMANubZA3Uz5g/edit?usp=sharing>`__.
```
Comment (Hwurzburg) [inline common/source/docs/common-esp8266-telemetry.rst:18]:
```suggestion
- The `Beyond Robotix - Kahuna <https://www.beyondrobotix.com/products/kahuna>`__ is a ready to use ESP8266 module that connects directly to an autopilot. The Kahuna is pre-flashed with a fork of ArduPilot firmware, has a JST-GH connector to connect easily to several off-the-shelf autopilots and has a u.fl connector for an external antenna. Further documentation for this unit is found `here <https://docs.google.com/document/d/1VyOsp9_q6dIAdYdWuDFcWoqqrNy_vbFMANubZA3Uz5g/edit?usp=sharing>`__.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-esp8266-telemetry.rst (hunk 1 above)
Comment (SamJCKnox) [reply common/source/docs/common-esp8266-telemetry.rst]: Thanks very much khancyr, I have made those changes. 
I don't have much experience updating the ardupilot wiki, but I use it a lot and am very keen to help contribute. My main interest is hardware design so I am interested in developing some hardware that can be used as part of the ardupilot ecosystem. I have several videos and figures I would like to add to this page, if I may, in due course.

**Comment by Hwurzburg** [comment]
squash and I will merge...thanks

**Comment by SamJCKnox** [comment]
@Hwurzburg Appologies, I'm not super familar with squashing - I followed [this](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together) but am unsure it has squashed correctly?

**Comment by Hwurzburg** [comment]
I can fix it for you but you have not checked the "allow other uses to make changes" box on the PR

**Pull request merged**